tests/baselines/reference/importCallExpressionGrammarError.js

```diff
@@ -10,8 +10,9 @@ const p2 = import();
 const p4 = import("pathToModule", "secondModule");
 
 //// [importCallExpressionGrammarError.js]
+var _a, _b;
 var a = ["./0"];
-Promise.resolve().then(() => require(...["PathModule"]));
-var p1 = Promise.resolve().then(() => require(...a));
+_a = (...["PathModule"]), Promise.resolve().then(() => require(_a));
+var p1 = (_b = (...a), Promise.resolve().then(() => require(_b)));

```

Comment:
This is invalid grammar, but the input is invalid grammar as well. I assume we can safely treat it as garbage in, garbage out?